### DISCUSSION TITLE
Icon weight incorrect when build with material-symbols-rounded

### DISCUSSION THREAD

**Discussion by altezza04** [quasarframework/quasar#16510] [General - Components / Directives / etc]
When using "material-symbols-rounded" web font, the Icon "weight" (thickness) looks correct during quasar dev, however when building, the Icon become very thin, what have I done wrong?

<img width="45" alt="Screenshot 2023-10-24 at 12 14 49 PM" src="https://github.com/quasarframework/quasar/assets/59946405/a2291568-cd26-42a5-a5a3-2115e6fc3d2a">
Icon in dev

<img width="63" alt="Screenshot 2023-10-24 at 12 14 41 PM" src="https://github.com/quasarframework/quasar/assets/59946405/1e8b5346-c9c7-414f-befc-1875809a2451">
Icon in build

**Comment by joaobatista2end** [comment]
Has anyone found a solution to this problem?
